packages/terra-date-picker/src/react-datepicker/PopupContainer.jsx

```diff
@@ -2,9 +2,21 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 const propTypes = {
+  /**
+   * Boolean prop for bounded height of container.
```

Comment:
There's not a strong standard for descriptions, but I think including `Boolean` and `prop` is slightly redundant.
 
My preference for booleans is a consistent phrasing.

> Whether or not... 

> Whether or not the height is bounded.

This isn't a set standard, feel free to phrase these however you feel appropriate.